src/SprykerEco/Client/Payone/PayoneClientInterface.php

```diff
@@ -163,4 +165,17 @@ public function sendAddressCheckRequest(QuoteTransfer $quoteTransfer): AddressCh
      * @return \Generated\Shared\Transfer\ConsumerScoreResponseTransfer
      */
     public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): ConsumerScoreResponseTransfer;
+
+    /**
+     * Specification:
+     * - Sends request to Payone to start session.
+     * - Gives a token in response for further requests.
```

Comment:
* Returns a client token in case of success, adds an error message otherwise.

Reply:
++